src/ae_kqueue.c

```diff
@@ -106,6 +110,9 @@ static void aeApiDelEvent(aeEventLoop *eventLoop, int fd, int mask) {
     }
 }
 
+#define EVENT_MASK_OFFSET(fd) (6 - (fd) % 4 * 2)
+#define EVENT_MASK_ENCODE(fd, mask) (((mask) & 0x3) << EVENT_MASK_OFFSET(fd))
+#define EVENT_MASK_DECODE(fd, mask) (((mask) >> EVENT_MASK_OFFSET(fd)) & 0x3);
```

Comment:
i think that inline functions or macros that do the whole reading and writing will be slightly cleaner (i.e. put the `/4` and `%4` in the same place).
it may mean that the `/4` and `%4` repeated twice, but at least they're in one place.

Reply:
Done.